src/GDGUkraine/templates/card.html

```diff
@@ -29,7 +30,56 @@
         div {
             text-align: justify;
         }
+        .approve-btn:hover {
+            background-color: black; 
+            color: white;
+        }
+        .btn {
+                        border-radius: 5px;
+            display: block;
+            padding: 10px 5px;
+            margin: 0 auto;
+            cursor: pointer;
+            width: 100px;
+            border: 0;
+            box-shadow: none;
+        }
+        .approve-btn {
+
+            color: black;
+            background-color: rgb(240,240,240);
+        }
+        .btn-success {
+    color: #fff;
+    background-color: #5cb85c;
+
+    border-color: #4cae4c;
+}
     </style>
+    <script type="text/javascript">
+        function approve_visit () {
+            $.ajax({
+              url: "http://localhost:8080/api/events/{{registration.id}}/check-in",
```

Comment:
In fact, you could generate the whole url, as follows:
https://github.com/GDG-Ukraine/gdg.org.ua/blob/master/src/GDGUkraine/templates/event.html#L57 from https://github.com/GDG-Ukraine/gdg.org.ua/blob/master/src/GDGUkraine/rest_controller.py#L598

`{{ url_for_routes('check-in', id=registration.id) }}` should work for you